admin/app/controllers/HealthCheck.scala

```diff
@@ -1,17 +1,9 @@
 package controllers
 
-import conf.AllGoodHealthcheckController
-import play.api.mvc.Action
+import conf.{AllGoodCachedHealthCheck, CachedHealthCheckLifeCycle}
 
-object HealthCheck extends AllGoodHealthcheckController(9001, "/login") {
+object HealthCheck extends AllGoodCachedHealthCheck(9001, "/login")
 
-  // Unlike other health checks, Admin is initially set to healthy, and may move into
-  // an unhealthy state, which it will remain in until terminated.
-  status.set(true)
-
-  override def healthcheck() = if (!isOk) {
-    Action(InternalServerError("JAXP00010001"))
-  } else {
-    super.healthcheck()
-  }
```

Comment:
The infamous `JAXP00010001` has been fixed: https://bugs.openjdk.java.net/browse/JDK-8028111

Note: we are running JDK 8u77